Index: src/main/java/ca/uhn/hl7v2/parser/XMLParser.java
===================================================================
--- src/main/java/ca/uhn/hl7v2/parser/XMLParser.java	(revision 513)
+++ src/main/java/ca/uhn/hl7v2/parser/XMLParser.java	(working copy)
@@ -105,6 +105,7 @@
      * @param theFactory custom factory to use for model class lookup 
      */
     public XMLParser(ModelClassFactory theFactory) {
+    	super(theFactory);
         parser = new DOMParser(new StandardParserConfiguration());
         try {
             parser.setFeature("http://apache.org/xml/features/dom/include-ignorable-whitespace", false);
